src/frontends/paddle/src/CMakeLists.txt

```diff
@@ -11,4 +11,4 @@ ov_add_frontend(NAME paddle
                 PROTOBUF_REQUIRED
                 PROTOBUF_LITE
                 FILEDESCRIPTION "FrontEnd to load and convert PaddlePaddle file format"
-                LINK_LIBRARIES openvino::util openvino::core::dev)
+                LINK_LIBRARIES openvino::util openvino::core::dev openvino::frontend::common_translators)
```

Comment:
maybe we need to link it in common place here:
https://github.com/openvinotoolkit/openvino/blob/1604ae539e17d6cd8676cec4976636d79b4f8bf5/cmake/developer_package/frontends/frontends.cmake#L231
instead of each FE manually?

Reply:
Done.